### PR TITLE
docs: Fixes, dagger call additions, remove ext core types

### PR DESCRIPTION
**Pull request by jpadams**

### CODE REVIEW

**Review by kpenfound — APPROVED** [review]
SHIPIT

**Comment by jpadams** [comment]
FYI @vikram-dagger

File: docs/current/labs/project-zenith.md [hunk 1]
```diff
@@ -175,7 +71,7 @@ mkdir potato/
 cd potato/
 
 # initialize Dagger module
-# NOTE: currently Go is the only supported SDK.
+# NOTE: currently only the Go SDK implements `dagger mod init`, but Python modules also work and are gaining features rapidly. Node.js modules are not yet available, but under development.
```
Comment (sipsma) [inline docs/current/labs/project-zenith.md:74]: `dagger mod init --sdk=python` does work, you just have to create a `pyproject.toml` and `src/main.py` first (it doesn't make those for you yet).

Just saw this was merged in the middle of writing my comment 😄 I think this is fine as is; we're meeting w/ Vikram tomorrow to talk more about docs, can easily follow-up more after that

Comment (helderco) [reply docs/current/labs/project-zenith.md:74]: I actually implemented this yesterday, just need to push! 🙂

Comment (helderco) [reply docs/current/labs/project-zenith.md:74]: - https://github.com/dagger/dagger/pull/5959

**Pull request merged**